OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory63.cs

```diff
@@ -0,0 +1,235 @@
+                    CastDurationMax = mem.CastDurationMax,
+                };
+                combatant.IsTargetable =
+                    (combatant.ModelStatus == ModelStatus.Visible)
+                    && ((combatant.Status == ObjectStatus.NormalActorStatus) || (combatant.Status == ObjectStatus.NormalSubActorStatus));
+                if (combatant.Type != ObjectType.PC && combatant.Type != ObjectType.Monster)
+                {
+                    // Other types have garbage memory for hp.
+                    combatant.CurrentHP = 0;
+                    combatant.MaxHP = 0;
+                }
+                return combatant;
+            }
+        }
+
+        [StructLayout(LayoutKind.Explicit)]
+        private unsafe struct CombatantMemory
+        {
+            public static int Size => Marshal.SizeOf(typeof(CombatantMemory));
+
+            // Unknown size, but this is the bytes up to the next field.
+            public const int NameBytes = 68;
+
+            public const int EffectCount = 60;
+            public const int EffectBytes = EffectMemory.Size * EffectCount;
+
+            [FieldOffset(0x30)]
+            public fixed byte Name[NameBytes];
+
+            [FieldOffset(0x74)]
+            public uint ID;
+
+            [FieldOffset(0x80)]
+            public uint BNpcID;
+
+            [FieldOffset(0x84)]
+            public uint OwnerID;
+
+            [FieldOffset(0x8C)]
+            public byte Type;
+
+            [FieldOffset(0x92)]
+            public byte EffectiveDistance;
+
+            [FieldOffset(0x94)]
+            public byte Status;
+
+            [FieldOffset(0xB0)]
+            public Single PosX;
+
+            [FieldOffset(0xB4)]
+            public Single PosY;
+
+            [FieldOffset(0xB8)]
+            public Single PosZ;
+
+            [FieldOffset(0xC0)]
+            public Single Heading;
+
+            [FieldOffset(0xD0)]
+            public Single Radius;
+
+            [FieldOffset(0x104)]
+            public int ModelStatus;
+
+            [FieldOffset(0x1C4)]
+            public int CurrentHP;
+
+            [FieldOffset(0x1C8)]
+            public int MaxHP;
+
+            [FieldOffset(0x1CC)]
+            public int CurrentMP;
+
+            [FieldOffset(0x1D0)]
+            public int MaxMP;
+
+            [FieldOffset(0x1D4)]
+            public ushort CurrentGP;
+
+            [FieldOffset(0x1D6)]
+            public ushort MaxGP;
+
+            [FieldOffset(0x1D8)]
+            public ushort CurrentCP;
+
+            [FieldOffset(0x1DA)]
+            public ushort MaxCP;
+
+            [FieldOffset(0x1E0)]
+            public byte Job;
+
+            [FieldOffset(0x1E1)]
+            public byte Level;
+
+            [FieldOffset(0xC80)]
+            public uint PCTargetID;
+
+            [FieldOffset(0x19C3)]
+            public byte MonsterType;
+
+            [FieldOffset(0x19DF)]
+            public byte AggressionStatus;
```

Comment:
Did you manually verify these two offsets? From the changes to https://github.com/aers/FFXIVClientStructs/blob/main/FFXIVClientStructs/FFXIV/Client/Game/Character/Character.cs, it looks like these two should be shifted +0x20 bytes, judging based on the bumped offset for VfxData pointers?

Reply:
Even with the +0x20 shift, I think these are both still incorrect, so maybe there's some extra shift inside of VfxData, or I am not sure what I am looking for.  I applied the shift and did a `GetCombatants` while attacking some mandragora with my chocobo out, and everything has MonsterType=0 and AggressionStatus=0.

I think it's probably worth landing this and following up to fix.  It'll break `GetTargetableEnemyList` but I think that's it (and is minor, imo).

Reply:
Yeah, I agree that it's worth landing this PR with a comment that those two offsets are still broken or something.

Re: VfxData, those are just 8-byte pointers, not full structs.

Obviously something's shifted around and is pushing everything back by 0x20 bytes further into the Combatant struct, but it's possible that they flat out moved these properties elsewhere as well.

Reply:
@valarnin Just to get this landed sooner rather than later, I'll commit my TODO messages here if you want to approve the PR.